### ISSUE TITLE
Is  div class="field"  wrapper needed for sm-input?

### ISSUE THREAD

**Issue by dzolnjan** [vladotesanovic/ngSemantic#39]
https://ng-semantic.herokuapp.com/#/elements/form

```
<div class="field">
	<sm-input label="Name" [control]="form.controls.nameControl" placeholder="Enter name..."></sm-input>
</div>
```

Form example uses that outer div wrapper but it seem that can be removed by looking at here:
https://github.com/vladotesanovic/ngSemantic/blob/master/ng-semantic/input/input.ts#L13

 Any particular reason why it is used in example?

**Comment by vladotesanovic** [comment]
Correct. I have problem to distinguish <input field being alone and being child of 'ui form' because if you see Semantic UI documentation, you will notice different semantic for those two scenarios:

For example:

When Input filed is displayed without 'ui form', it is wrapped with 'ui input' and errors goes on that element: 'ui input error' 
`<div class="ui input error">
  <input type="text" placeholder="Search...">
</div>`

In scenario with 'ui form' it goes like this:
`<div class="ui form">
  <div class="two fields">
    <div class="field error">
      <label>First Name</label>
      <input placeholder="First Name" type="text">
    </div>
....`

My goal is to make one component that can be applied to all scenarios.
